src/Bridges/Bridges.jl

```diff
@@ -59,6 +61,26 @@ end
 
 include("vectorizebridge.jl")
 @bridge Vectorize VectorizeBridge () (MOI.EqualTo, MOI.LessThan, MOI.GreaterThan,) () () (MOI.SingleVariable,) (MOI.ScalarAffineFunction, MOI.ScalarQuadraticFunction) () ()
+include("slackbridge.jl")
+@bridge ScalarSlack ScalarSlackBridge () (MOI.Interval, MOI.LessThan, MOI.GreaterThan) () () () (MOI.ScalarAffineFunction, MOI.ScalarQuadraticFunction) () ()
+@bridge(VectorSlack, VectorSlackBridge, 
+(),
```

Comment:
Indent and merge some lines if it fits 80 chars. Having to list all sets is not ideal but the need will be removed by light bridges